pages/common/keepassxc-cli.md

```diff
@@ -0,0 +1,36 @@
+# keepassxc-cli
+
+> Command Line interface for the keepassxc app.
+> More information: <https://keepassxc.org/>.
+
+- Search entries:
+
+`keepassxc-cli lookup {{database_file}} {{name}}`
+
+- List a folder:
```

Comment:
```suggestion
- List the content a folder:
```
I hope that's correct. Also, is this a real folder and if yes do you have to specify a full path?

Reply:
Shouldn't that be contents

Reply:
Well, it doesn't really matter — they're actually mostly synonymous.  I think `contents` is more generally used in the context, so maybe let's go with that.